CIP-0067/registry.json

```diff
@@ -9,6 +9,11 @@
     "class": "NFT",
     "description": "CIP-0068 - Datum Metadata Standard (222 sub standard)"
   },
+  {
+    "asset_name_label": 314,
+    "class": "Virtual_SubHandle",
```

Comment:
I'm not sure if this should be accepted as a valid `class`. What does this represent?

Reply:
An upcoming feature of ADA Handles is SubHandles. A root Handle owner can choose to sell NFT SubHandles that essentially behave just like Handles today. They are actual NFTs that the root Handle owner loses control over the moment it mints and is sent to the new owner.

Virtual SubHandles are SubHandles that the root Handle owner retains control over and can revoke or reassign. This means, this particular class of Handle will need to resolve addresses by UTxO datum in the token.

This is to support more corporate or enterprise use cases where SubHandles may be temporarily issued to employees or representatives of the larger entity. It can also support other smaller business use cases of SubHandle "renting".

Virtual SubHandles will have nearly all the same personalization characteristics of Handles.

Reply:
With the (222) Handles - the personalization data will be housed by the (100) Reference Token.
With the (314) Virtual SubHandles - the personalization data will be housed in the token itself.